reference/3.0/Microsoft.PowerShell.Management/Start-Process.md

```diff
@@ -1,142 +1,149 @@
+This example shows how to find the verbs that can be used when starting a process. The available
+verbs are determined by the filename extension of the file that runs in the process.
 
-### Example 7: Specifying arguments to the process
+```powershell
+$startExe = New-Object System.Diagnostics.ProcessStartInfo -Args PowerShell.exe
+$startExe.verbs
 ```
-PS C:\> Start-Process -FilePath "$env:comspec" -ArgumentList "/c dir `"%systemdrive%\program files`""
-PS C:\> Start-Process -FilePath "$env:comspec" -ArgumentList "/c","dir","`"%systemdrive%\program files`""
+
+```Output
+open
+runas
 ```
-Both commands start the Windows command interpreter, issueing a dir command on the 'Program Files' folder.
-Because this foldername contains a space, the value needs surrounded with escaped quotes.
+
+The example uses `New-Object` to create a **System.Diagnostics.ProcessStartInfo** object for
+**PowerShell.exe**, the file that runs in the PowerShell process. The **Verbs** property of the
+**ProcessStartInfo** object shows that you can use the **Open** and **RunAs** verbs with
+**PowerShell.exe**, or with any process that runs a .exe file.
+
+### Example 7: Specifying arguments to the process
+
+Both commands start the Windows command interpreter, issuing a dir command on the 'Program Files'
+folder. Because this foldername contains a space, the value needs surrounded with escaped quotes.
 Note that the first command specifies a string as ArgumentList. The second command a string array.
 
+```powershell
+Start-Process -FilePath "$env:comspec" -ArgumentList "/c dir `"%systemdrive%\program files`""
+Start-Process -FilePath "$env:comspec" -ArgumentList "/c","dir","`"%systemdrive%\program files`""
+```
+
 ## PARAMETERS
 
 ### -ArgumentList
-Specifies parameters or parameter values to use when starting the process.
-If parameters or parameter values contain a space, they need surrounded with escaped double quotes.
-The parameter name ("ArgumentList") is optional.
+
+Specifies parameters or parameter values to use when this cmdlet starts the process. If parameters
+or parameter values contain a space, they need surrounded with escaped double quotes.
 
 ```yaml
 Type: String[]
 Parameter Sets: (All)
 Aliases: Args
 
 Required: False
-Position: 2
+Position: 1
 Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
 ### -Credential
-Specifies a user account that has permission to perform this action.
-Type a user-name, such as "User01" or "Domain01\User01", or enter a PSCredential object, such as one from the Get-Credential cmdlet.
-By default, the cmdlet uses the credentials of the current user.
+
+Specifies a user account that has permission to perform this action. Type a user name, such as
+User01 or Domain01\User01, or enter a **PSCredential** object, such as one from the `Get-Credential`
+cmdlet. By default, the cmdlet uses the credentials of the current user.
```

Comment:
Is this still true?  The default value below was changed to `None`.

Reply:
The default value is None. The default behavior when the value is None is to use the current user.